FiegeIntegration/Overview/01_General.md

```diff
@@ -0,0 +1,27 @@
+# General information
+
+The *Fiege* driver seamlessly connects the *Actindo Core 1 Platform* and the Fiege fulfillment system. Actindo exports automatically all relevant product data as soon as a delivery note is generated. In turn, all shipment data provided by Fiege can be checked and monitored centrally in the *Fulfillment* module. 
```

Comment:
```suggestion
The *Fiege* driver seamlessly connects the *Actindo Core1 Platform* and the Fiege fulfillment system. Actindo exports automatically all relevant product data as soon as a delivery note is generated. In turn, all shipment data provided by Fiege can be checked and monitored centrally in the *Fulfillment* module. 
```
delivery note oder dispatch note? Meines Wissens ist es in Fulfillment die dispatch note, in OMS die delivery note